src/iden3comm/types/protocol/auth.ts

```diff
@@ -48,5 +48,16 @@ export type ZeroKnowledgeProofRequest = {
 export type ZeroKnowledgeProofResponse = {
   id: number;
   circuitId: string;
-  vp?: object;
+  vp?: VerifiablePresentation;
 } & ZKProof;
+
+/** VerifiablePresentation represents structure of Verifiable Presentation */
+export type VerifiablePresentation = {
+  '@context': string[];
```

Comment:
JSONObject[] ( but not bool and integer). Members of array should be string | object . 
Also context can be string.

Reply:
fixed